pysubs2/substation.py

```diff
@@ -184,22 +184,22 @@ def string_to_field(f, v):
             elif inside_info_section or inside_aegisub_section:
                 if line.startswith(";"): continue # skip comments
                 try:
-                    k, v = line.split(": ", 1)
+                    k, v = line.split(":", 1)
                     if inside_info_section:
-                        subs.info[k] = v
+                        subs.info[k] = v.strip()
                     elif inside_aegisub_section:
                         subs.aegisub_project[k] = v
```

Comment:
```suggestion
                         subs.aegisub_project[k] = v.strip()
```
This should fix the broken test.